core/extension/gdextension_interface.cpp

```diff
@@ -1625,6 +1710,7 @@ void gdextension_setup_interface() {
 	REGISTER_INTERFACE_FUNC(variant_can_convert_strict);
 	REGISTER_INTERFACE_FUNC(get_variant_from_type_constructor);
 	REGISTER_INTERFACE_FUNC(get_variant_to_type_constructor);
+	REGISTER_INTERFACE_FUNC(variant_get_ptr_internal);
 	REGISTER_INTERFACE_FUNC(variant_get_ptr_operator_evaluator);
 	REGISTER_INTERFACE_FUNC(variant_get_ptr_builtin_method);
 	REGISTER_INTERFACE_FUNC(variant_get_ptr_constructor);
```

Comment:
(Bikeshedding) the existing `variant_get_ptr_*` names indicate that a function is returned, with names such as "evaluator", "method" and "constructor":

```
variant_get_ptr_operator_evaluator
variant_get_ptr_builtin_method
variant_get_ptr_constructor
```

The name "internal" doesn't seem to fit that scheme. Maybe `variant_get_ptr_internal_accessor` or something?

Just something to indicate it doesn't return the internal pointer, but a function that retrieves it.

Reply:
I've renamed the function to `variant_get_ptr_internal_getter`. The use of 'getter' seems to already be somewhat established in the interface.